machine-config-daemon.spec

```diff
@@ -28,8 +28,10 @@ env VERSION_OVERRIDE=%{version} SOURCE_GIT_COMMIT=%{commit} make daemon
 
 %install
 install -D -m 0755 _output/linux/*/%{name} $RPM_BUILD_ROOT/usr/libexec/%{name}
+install -D -m 0755 cmd/machine-config-daemon/pivot.sh $RPM_BUILD_ROOT/%{_bindir}/pivot
 
 %files
 %license LICENSE
 %doc README.md
 %{_libexecdir}/%{name}
+%{_bindir}/pivot
```

Comment:
isn't this conflicting with https://github.com/openshift/machine-config-operator/blob/master/pkg/daemon/rpm-ostree.go#L132

Reply:
Uhm, this slipped through, @cgwalters can you take a super fast look?

Reply:
You're totally right but because [we aren't auto-building the RPM yet](https://jira.coreos.com/browse/ART-640) we have time to do a PR to drop the compat code which I think we can do now that [a RHCOS bootimage update](https://github.com/openshift/installer/pull/1895) landed.